ipq806x: scaling improvement and dts rework

Ansuel opened the pull request:
This patchset rework the dts to a better pinmux definition.
Modify the L2 scaling patch to reflect the original QSDK scaling, add
support for l2 regulator voltage scaling and l2 frequency scaling based
on cpu frequency.
Add fab-scaling support to switch to idle frequency when the router is
idle.
Fix a broken patch that introduce voltage scaling but never actually use it.

Signed-off-by: Ansuel Smith <ansuelsmth@gmail.com>

Ansuel committed:
ipq806x: opp/core: fix broken patch Handle opp voltage adjust

The opp_notifier introduced with patch 0054-Handle-OPP-voltage adjust
for some reason missed the actual registration in the opp struct,
resulting in never being registred.
This was present in kernel 4.9 patchset but dropped in the transition
 to 4.14. Reintroduce this and fix patch 0055 about L2 cache scaling.

Signed-off-by: Ansuel Smith <ansuelsmth@gmail.com>

Ansuel committed:
ipq806x: reorganize nand_pins mux definition

It's wrong set the mux to bias-disable. The best way to
do this is by creating a separate group and disable the
specific pins.

By documentation, any subgroup with no bias definition
is ignored so the mux definition is useless.
Rework the definition by sremoving the mux subgroup and
set the remaining subgroup with the mux function and
drive-strength

Signed-off-by: Ansuel Smith <ansuelsmth@gmail.com>

Ansuel committed:
ipq806x: add fab scaling support

Add fab scaling support and dtsi definition

Signed-off-by: Ansuel Smith <ansuelsmth@gmail.com>

pkgadd commented:
Seems to work fine on my nbg6817/ ipq8064.

`Tested-by: Stefan Lippers-Hollmann <s.l-h@gmx.de>`

shelterx commented:
No apparent issues here either, been running this for almost 2 days now.

hnyman commented:
Seems to work ok in ipq8065/R7800

facboy commented:
i've been running the various iterations of this for a week or so on an R7800, no problems so far.

Noltari commented:
Latest version is causing errors on my Netgear R7800:
`[    0.000000] Booting Linux on physical CPU 0x0
[    0.000000] Linux version 4.19.93 (noltari@skynet) (gcc version 8.3.0 (OpenWrt GCC 8.3.0 r11536-cde8c2f2fb)) #0 SMP Sun Jan 5 12:21:49 2020
[    0.000000] CPU: ARMv7 Processor [512f04d0] revision 0 (ARMv7), cr=10c5787d
[    0.000000] CPU: div instructions available: patching division code
[    0.000000] CPU: PIPT / VIPT nonaliasing data cache, PIPT instruction cache
[    0.000000] OF: fdt: Machine model: Netgear Nighthawk X4S R7800
[    0.000000] Memory policy: Data cache writealloc
[    0.000000] On node 0 totalpages: 122880
[    0.000000]   Normal zone: 1080 pages used for memmap
[    0.000000]   Normal zone: 0 pages reserved
[    0.000000]   Normal zone: 122880 pages, LIFO batch:31
[    0.000000] random: get_random_bytes called from start_kernel+0x80/0x47c with crng_init=0
[    0.000000] percpu: Embedded 15 pages/cpu s29964 r8192 d23284 u61440
[    0.000000] pcpu-alloc: s29964 r8192 d23284 u61440 alloc=15*4096
[    0.000000] pcpu-alloc: [0] 0 [0] 1 
[    0.000000] Built 1 zonelists, mobility grouping on.  Total pages: 121800
[    0.000000] Kernel command line: 
[    0.000000] Bootloader command line (ignored): console=ttyHSL1,115200n8
[    0.000000] Dentry cache hash table entries: 65536 (order: 6, 262144 bytes)
[    0.000000] Inode-cache hash table entries: 32768 (order: 5, 131072 bytes)
[    0.000000] Memory: 475876K/491520K available (5322K kernel code, 180K rwdata, 1356K rodata, 1024K init, 228K bss, 15644K reserved, 0K cma-reserved, 0K highmem)
[    0.000000] Virtual kernel memory layout:
[    0.000000]     vector  : 0xffff0000 - 0xffff1000   (   4 kB)
[    0.000000]     fixmap  : 0xffc00000 - 0xfff00000   (3072 kB)
[    0.000000]     vmalloc : 0xde800000 - 0xff800000   ( 528 MB)
[    0.000000]     lowmem  : 0xc0000000 - 0xde000000   ( 480 MB)
[    0.000000]     pkmap   : 0xbfe00000 - 0xc0000000   (   2 MB)
[    0.000000]     modules : 0xbf000000 - 0xbfe00000   (  14 MB)
[    0.000000]       .text : 0x(ptrval) - 0x(ptrval)   (6315 kB)
[    0.000000]       .init : 0x(ptrval) - 0x(ptrval)   (1024 kB)
[    0.000000]       .data : 0x(ptrval) - 0x(ptrval)   ( 181 kB)
[    0.000000]        .bss : 0x(ptrval) - 0x(ptrval)   ( 229 kB)
[    0.000000] SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=2, Nodes=1
[    0.000000] rcu: Hierarchical RCU implementation.
[    0.000000] NR_IRQS: 16, nr_irqs: 16, preallocated irqs: 16
[    0.000000] clocksource: dg_timer: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 305801671480 ns
[    0.000007] sched_clock: 32 bits at 6MHz, resolution 160ns, wraps every 343597383600ns
[    0.000021] Switching to timer-based delay loop, resolution 160ns
[    0.000235] Calibrating delay loop (skipped), value calculated using timer frequency.. 12.50 BogoMIPS (lpj=62500)
[    0.000256] pid_max: default: 32768 minimum: 301
[    0.000421] Mount-cache hash table entries: 1024 (order: 0, 4096 bytes)
[    0.000440] Mountpoint-cache hash table entries: 1024 (order: 0, 4096 bytes)
[    0.001316] CPU: Testing write buffer coherency: ok
[    0.002398] Setting up static identity map for 0x42300000 - 0x42300060
[    0.002565] rcu: Hierarchical SRCU implementation.
[    0.003383] smp: Bringing up secondary CPUs ...
[    0.005288] smp: Brought up 1 node, 2 CPUs
[    0.005303] SMP: Total of 2 processors activated (25.00 BogoMIPS).
[    0.005314] CPU: All CPU(s) started in SVC mode.
[    0.015704] VFP support v0.3: implementor 51 architecture 64 part 4d variant 2 rev 0
[    0.015852] clocksource: jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 19112604462750000 ns
[    0.015879] futex hash table entries: 512 (order: 3, 32768 bytes)
[    0.016039] pinctrl core: initialized pinctrl subsystem
[    0.017133] NET: Registered protocol family 16
[    0.017420] DMA: preallocated 256 KiB pool for atomic coherent allocations
[    0.018716] cpuidle: using governor ladder
[    0.018780] cpuidle: using governor menu
[    0.041914] usbcore: registered new interface driver usbfs
[    0.041993] usbcore: registered new interface driver hub
[    0.042080] usbcore: registered new device driver usb
[    0.042146] pps_core: LinuxPPS API ver. 1 registered
[    0.042160] pps_core: Software ver. 5.3.6 - Copyright 2005-2007 Rodolfo Giometti <giometti@linux.it>
[    0.042196] PTP clock support registered
[    0.043748] clocksource: Switched to clocksource dg_timer
[    0.046839] NET: Registered protocol family 2
[    0.047374] tcp_listen_portaddr_hash hash table entries: 512 (order: 0, 6144 bytes)
[    0.047413] TCP established hash table entries: 4096 (order: 2, 16384 bytes)
[    0.047450] TCP bind hash table entries: 4096 (order: 3, 32768 bytes)
[    0.047503] TCP: Hash tables configured (established 4096 bind 4096)
[    0.047603] UDP hash table entries: 256 (order: 1, 8192 bytes)
[    0.047636] UDP-Lite hash table entries: 256 (order: 1, 8192 bytes)
[    0.047864] NET: Registered protocol family 1
[    0.047912] PCI: CLS 0 bytes, default 64
[    0.049108] No memory allocated for crashlog
[    0.049378] workingset: timestamp_bits=14 max_order=17 bucket_order=3
[    0.055460] squashfs: version 4.0 (2009/01/31) Phillip Lougher
[    0.055476] jffs2: version 2.2 (NAND) (SUMMARY) (LZMA) (RTIME) (CMODE_PRIORITY) (c) 2001-2006 Red Hat, Inc.
[    0.064345] io scheduler noop registered
[    0.064362] io scheduler deadline registered (default)
[    0.066307] qcom-pcie 1b500000.pci: 1b500000.pci supply vdda not found, using dummy regulator
[    0.066432] qcom-pcie 1b500000.pci: Linked as a consumer to regulator.0
[    0.066459] qcom-pcie 1b500000.pci: 1b500000.pci supply vdda_phy not found, using dummy regulator
[    0.066573] qcom-pcie 1b500000.pci: 1b500000.pci supply vdda_refclk not found, using dummy regulator
[    0.066831] qcom-pcie 1b500000.pci: host bridge /soc/pci@1b500000 ranges:
[    0.066856] qcom-pcie 1b500000.pci: Parsing ranges property...
[    0.066888] qcom-pcie 1b500000.pci:    IO 0x0fe00000..0x0fefffff -> 0x0fe00000
[    0.066920] qcom-pcie 1b500000.pci:   MEM 0x08000000..0x0fdfffff -> 0x08000000
[    0.289204] qcom-pcie 1b500000.pci: Link up
[    0.289364] qcom-pcie 1b500000.pci: PCI host bridge to bus 0000:00
[    0.289386] pci_bus 0000:00: root bus resource [bus 00-ff]
[    0.289405] pci_bus 0000:00: root bus resource [io  0x0000-0xfffff] (bus address [0xfe00000-0xfefffff])
[    0.289420] pci_bus 0000:00: root bus resource [mem 0x08000000-0x0fdfffff]
[    0.289435] pci_bus 0000:00: scanning bus
[    0.289484] pci 0000:00:00.0: [17cb:0101] type 01 class 0x060400
[    0.289623] pci 0000:00:00.0: supports D1
[    0.289632] pci 0000:00:00.0: PME# supported from D0 D1 D3hot
[    0.289650] pci 0000:00:00.0: PME# disabled
[    0.293685] pci_bus 0000:00: fixups for bus
[    0.293709] PCI: bus0: Fast back to back transfers disabled
[    0.293730] pci 0000:00:00.0: scanning [bus 01-ff] behind bridge, pass 0
[    0.293867] pci_bus 0000:01: scanning bus
[    0.294152] pci 0000:01:00.0: [168c:0046] type 00 class 0x028000
[    0.294500] pci 0000:01:00.0: reg 0x10: [mem 0x00000000-0x001fffff 64bit]
[    0.295595] pci 0000:01:00.0: PME# supported from D0 D3hot D3cold
[    0.295635] pci 0000:01:00.0: PME# disabled
[    0.299795] pci_bus 0000:01: fixups for bus
[    0.299867] PCI: bus1: Fast back to back transfers disabled
[    0.299884] pci_bus 0000:01: bus scan returning with max=01
[    0.299901] pci 0000:00:00.0: scanning [bus 01-ff] behind bridge, pass 1
[    0.299917] pci_bus 0000:00: bus scan returning with max=ff
[    0.299965] pci 0000:00:00.0: BAR 8: assigned [mem 0x08000000-0x081fffff]
[    0.299989] pci 0000:01:00.0: BAR 0: assigned [mem 0x08000000-0x081fffff 64bit]
[    0.300117] pci 0000:00:00.0: PCI bridge to [bus 01-ff]
[    0.300140] pci 0000:00:00.0:   bridge window [mem 0x08000000-0x081fffff]
[    0.300347] pcieport 0000:00:00.0: assign IRQ: got 35
[    0.300736] pcieport 0000:00:00.0: AER enabled with IRQ 36
[    0.301225] qcom-pcie 1b700000.pci: 1b700000.pci supply vdda not found, using dummy regulator
[    0.301325] qcom-pcie 1b700000.pci: Linked as a consumer to regulator.0
[    0.301351] qcom-pcie 1b700000.pci: 1b700000.pci supply vdda_phy not found, using dummy regulator
[    0.301463] qcom-pcie 1b700000.pci: 1b700000.pci supply vdda_refclk not found, using dummy regulator
[    0.301735] qcom-pcie 1b700000.pci: host bridge /soc/pci@1b700000 ranges:
[    0.301759] qcom-pcie 1b700000.pci: Parsing ranges property...
[    0.301788] qcom-pcie 1b700000.pci:    IO 0x31e00000..0x31efffff -> 0x31e00000
[    0.301815] qcom-pcie 1b700000.pci:   MEM 0x2e000000..0x31dfffff -> 0x2e000000
[    0.531279] qcom-pcie 1b700000.pci: Link up
[    0.531423] qcom-pcie 1b700000.pci: PCI host bridge to bus 0001:00
[    0.531446] pci_bus 0001:00: root bus resource [bus 00-ff]
[    0.531460] pci_bus 0001:00: root bus resource [mem 0x2e000000-0x31dfffff]
[    0.531473] pci_bus 0001:00: scanning bus
[    0.531514] pci 0001:00:00.0: [17cb:0101] type 01 class 0x060400
[    0.531635] pci 0001:00:00.0: supports D1
[    0.531644] pci 0001:00:00.0: PME# supported from D0 D1 D3hot
[    0.531657] pci 0001:00:00.0: PME# disabled
[    0.535715] pci_bus 0001:00: fixups for bus
[    0.535736] PCI: bus0: Fast back to back transfers disabled
[    0.535754] pci 0001:00:00.0: scanning [bus 01-ff] behind bridge, pass 0
[    0.535888] pci_bus 0001:01: scanning bus
[    0.536083] pci 0001:01:00.0: [168c:0046] type 00 class 0x028000
[    0.536452] pci 0001:01:00.0: reg 0x10: [mem 0x00000000-0x001fffff 64bit]
[    0.537626] pci 0001:01:00.0: PME# supported from D0 D3hot D3cold
[    0.537668] pci 0001:01:00.0: PME# disabled
[    0.537904] pci 0001:01:00.0: 2.000 Gb/s available PCIe bandwidth, limited by 2.5 GT/s x1 link at 0001:00:00.0 (capable of 4.000 Gb/s with 5 GT/s x1 link)
[    0.541859] pci_bus 0001:01: fixups for bus
[    0.541936] PCI: bus1: Fast back to back transfers disabled
[    0.541952] pci_bus 0001:01: bus scan returning with max=01
[    0.541969] pci 0001:00:00.0: scanning [bus 01-ff] behind bridge, pass 1
[    0.541985] pci_bus 0001:00: bus scan returning with max=ff
[    0.542028] pci 0001:00:00.0: BAR 8: assigned [mem 0x2e000000-0x2e1fffff]
[    0.542050] pci 0001:01:00.0: BAR 0: assigned [mem 0x2e000000-0x2e1fffff 64bit]
[    0.542182] pci 0001:00:00.0: PCI bridge to [bus 01-ff]
[    0.542204] pci 0001:00:00.0:   bridge window [mem 0x2e000000-0x2e1fffff]
[    0.542395] pcieport 0001:00:00.0: assign IRQ: got 37
[    0.542731] pcieport 0001:00:00.0: AER enabled with IRQ 38
[    0.545202] L2 @ QSB rate. Forcing new rate.
[    0.545415] L2 @ 384000 KHz
[    0.545591] CPU0 @ 800000 KHz
[    0.545604] CPU1 @ QSB rate. Forcing new rate.
[    0.545732] CPU1 @ 384000 KHz
[    0.549251] gsbi 16300000.gsbi: GSBI port protocol: 6 crci: 0
[    0.551344] Serial: 8250/16550 driver, 2 ports, IRQ sharing disabled
[    0.552128] msm_serial 16340000.serial: msm_serial: detected port #0
[    0.552178] msm_serial 16340000.serial: uartclk = 7372800
[    0.552247] 16340000.serial: ttyMSM0 at MMIO 0x16340000 (irq = 39, base_baud = 460800) is a MSM
[    0.552284] msm_serial: console setup on port #0
[    1.306240] console [ttyMSM0] enabled
[    1.311197] msm_serial: driver initialized
[    1.319458] loop: module loaded
[    1.321306] nand: device found, Manufacturer ID: 0x2c, Chip ID: 0xa1
[    1.321579] nand: Micron MT29F1G08ABBEAH4
[    1.328284] nand: 128 MiB, SLC, erase size: 128 KiB, page size: 2048, OOB size: 64
[    1.332176] 8 fixed-partitions partitions found on MTD device qcom_nand.0
[    1.339642] Creating 8 MTD partitions on "qcom_nand.0":
[    1.346496] 0x000000000000-0x000000c80000 : "qcadata"
[    1.356692] random: fast init done
[    1.372684] 0x000000c80000-0x000001180000 : "APPSBL"
[    1.381664] 0x000001180000-0x000001200000 : "APPSBLENV"
[    1.383250] 0x000001200000-0x000001340000 : "art"
[    1.388591] 0x000001340000-0x000001480000 : "artbak"
[    1.393330] 0x000001480000-0x000001880000 : "kernel"
[    1.402904] 0x000001880000-0x000007900000 : "ubi"
[    1.561160] 0x000007900000-0x000008000000 : "reserve"
[    1.574670] libphy: GPIO Bitbanged MDIO: probed
[    1.595872] switch0: Atheros AR8337 rev. 2 switch registered on gpio-0
[    2.455955] ar8327: qca,phy-rgmii-en is not specified
[    2.456324] libphy: Fixed MDIO Bus: probed
[    2.462043] ipq806x-gmac-dwmac 37200000.ethernet: PTP uses main clock
[    2.464787] ipq806x-gmac-dwmac 37200000.ethernet: User ID: 0x10, Synopsys ID: 0x37
[    2.470503] ipq806x-gmac-dwmac 37200000.ethernet: 	DWMAC1000
[    2.478055] ipq806x-gmac-dwmac 37200000.ethernet: DMA HW capability register supported
[    2.483781] ipq806x-gmac-dwmac 37200000.ethernet: RX Checksum Offload Engine supported
[    2.491592] ipq806x-gmac-dwmac 37200000.ethernet: COE Type 2
[    2.499460] ipq806x-gmac-dwmac 37200000.ethernet: TX Checksum insertion supported
[    2.505301] ipq806x-gmac-dwmac 37200000.ethernet: Wake-Up On Lan supported
[    2.512597] ipq806x-gmac-dwmac 37200000.ethernet: Enhanced/Alternate descriptors
[    2.519453] ipq806x-gmac-dwmac 37200000.ethernet: Enabled extended descriptors
[    2.526983] ipq806x-gmac-dwmac 37200000.ethernet: Ring mode enabled
[    2.533949] ipq806x-gmac-dwmac 37200000.ethernet: Enable RX Mitigation via HW Watchdog Timer
[    2.541841] ipq806x-gmac-dwmac 37400000.ethernet: PTP uses main clock
[    2.549202] ipq806x-gmac-dwmac 37400000.ethernet: User ID: 0x10, Synopsys ID: 0x37
[    2.555255] ipq806x-gmac-dwmac 37400000.ethernet: 	DWMAC1000
[    2.562594] ipq806x-gmac-dwmac 37400000.ethernet: DMA HW capability register supported
[    2.568503] ipq806x-gmac-dwmac 37400000.ethernet: RX Checksum Offload Engine supported
[    2.576217] ipq806x-gmac-dwmac 37400000.ethernet: COE Type 2
[    2.584120] ipq806x-gmac-dwmac 37400000.ethernet: TX Checksum insertion supported
[    2.589852] ipq806x-gmac-dwmac 37400000.ethernet: Wake-Up On Lan supported
[    2.597303] ipq806x-gmac-dwmac 37400000.ethernet: Enhanced/Alternate descriptors
[    2.604080] ipq806x-gmac-dwmac 37400000.ethernet: Enabled extended descriptors
[    2.611564] ipq806x-gmac-dwmac 37400000.ethernet: Ring mode enabled
[    2.618682] ipq806x-gmac-dwmac 37400000.ethernet: Enable RX Mitigation via HW Watchdog Timer
[    2.625510] i2c /dev entries driver
[    2.636569] cpuidle: enable-method property 'qcom,kpss-acc-v1' found operations
[    2.636796] cpuidle: enable-method property 'qcom,kpss-acc-v1' found operations
[    2.644890] sdhci: Secure Digital Host Controller Interface driver
[    2.651222] sdhci: Copyright(c) Pierre Ossman
[    2.657596] sdhci-pltfm: SDHCI platform and OF driver helper
[    2.665260] NET: Registered protocol family 10
[    2.669841] Segment Routing with IPv6
[    2.672024] NET: Registered protocol family 17
[    2.676047] 8021q: 802.1Q VLAN Support v1.8
[    2.680004] Registering SWP/SWPB emulation handler
[    2.709771] qcom_rpm 108000.rpm: RPM firmware 3.0.16777364
[    2.725326] s1a: supplied by regulator-dummy
[    2.725472] s1a: Bringing 0uV into 1050000-1050000uV
[    2.729088] s1b: supplied by regulator-dummy
[    2.733745] s1b: Bringing 0uV into 1050000-1050000uV
[    2.738372] s2a: supplied by regulator-dummy
[    2.742915] s2a: Bringing 0uV into 775000-775000uV
[    2.747549] s2b: supplied by regulator-dummy
[    2.751788] s2b: Bringing 0uV into 775000-775000uV
[    2.758364] Speed bin: 0
[    2.760695] PVS bin: 4
[    2.763888] cpu cpu0: Linked as a consumer to regulator.4
[    2.765786] cpu cpu0: Dropping the link to regulator.4
[    2.771351] cpu cpu0: Linked as a consumer to regulator.4
[    2.778185] cpu cpu0: Linked as a consumer to regulator.2
[    2.782096] cpu cpu1: Linked as a consumer to regulator.5
[    2.789072] cpu cpu1: Linked as a consumer to regulator.2
[    2.804397] UBI: auto-attach mtd6
[    2.804417] ubi0: attaching mtd6
[    2.902846] random: crng init done
[    3.534974] ubi0: scanning is finished
[    3.544429] ubi0: attached mtd6 (name "ubi", size 96 MiB)
[    3.544448] ubi0: PEB size: 131072 bytes (128 KiB), LEB size: 126976 bytes
[    3.548797] ubi0: min./max. I/O unit sizes: 2048/2048, sub-page size 2048
[    3.555649] ubi0: VID header offset: 2048 (aligned 2048), data offset: 4096
[    3.562424] ubi0: good PEBs: 772, bad PEBs: 0, corrupted PEBs: 0
[    3.569278] ubi0: user volume: 2, internal volumes: 1, max. volumes count: 128
[    3.575514] ubi0: max/mean erase counter: 4/1, WL threshold: 4096, image sequence number: 1164284368
[    3.582481] ubi0: available PEBs: 0, total reserved PEBs: 772, PEBs reserved for bad PEB handling: 20
[    3.591896] ubi0: background thread "ubi_bgt0d" started, PID 63
[    3.592515] block ubiblock0_0: created from ubi0:0(rootfs)
[    3.606663] ubiblock: device ubiblock0_0 (rootfs) set to be root filesystem
[    3.612167] hctosys: unable to open rtc device (rtc0)
[    3.623763] VFS: Mounted root (squashfs filesystem) readonly on device 254:0.
[    3.625352] Freeing unused kernel memory: 1024K
[    3.654317] Run /sbin/init as init process
[    3.925953] init: Console is alive
[    3.926072] init: - watchdog -
[    4.036326] s1a: Restricting voltage, 1150000-1102500uV
[    4.861391] kmodloader: loading kernel modules from /etc/modules-boot.d/*
[    4.877811] ehci_hcd: USB 2.0 'Enhanced' Host Controller (EHCI) Driver
[    4.880313] SCSI subsystem initialized
[    4.886456] libata version 3.00 loaded.
[    4.888400] ahci 29000000.sata: forcing port_map 0x0 -> 0x1
[    4.888442] ahci 29000000.sata: AHCI 0001.0300 32 slots 1 ports 6 Gbps 0x1 impl platform mode
[    4.892776] ahci 29000000.sata: flags: ncq sntf pm led clo only pmp pio slum part ccc apst 
[    4.901984] scsi host0: ahci
[    4.909864] ata1: SATA max UDMA/133 mmio [mem 0x29000000-0x2900017f] port 0x100 irq 26
[    5.245059] ata1: SATA link down (SStatus 0 SControl 300)
[    5.250433] ehci-platform: EHCI generic platform driver
[    5.255102] ohci_hcd: USB 1.1 'Open' Host Controller (OHCI) Driver
[    5.255779] ohci-platform: OHCI generic platform driver
[    5.265860] dwc3 11000000.dwc3: Failed to get clk 'ref': -2
[    5.270971] s1a: Restricting voltage, 1150000-1102500uV
[    5.314847] dwc3 10000000.dwc3: Failed to get clk 'ref': -2
[    5.376409] xhci-hcd xhci-hcd.0.auto: xHCI Host Controller
[    5.376459] xhci-hcd xhci-hcd.0.auto: new USB bus registered, assigned bus number 1
[    5.380965] xhci-hcd xhci-hcd.0.auto: hcc params 0x0228f065 hci version 0x100 quirks 0x0000000002010010
[    5.388486] xhci-hcd xhci-hcd.0.auto: irq 43, io mem 0x11000000
[    5.398204] s1a: Restricting voltage, 1150000-1102500uV
[    5.404095] hub 1-0:1.0: USB hub found
[    5.408868] hub 1-0:1.0: 1 port detected
[    5.412848] xhci-hcd xhci-hcd.0.auto: xHCI Host Controller
[    5.416795] xhci-hcd xhci-hcd.0.auto: new USB bus registered, assigned bus number 2
[    5.422020] xhci-hcd xhci-hcd.0.auto: Host supports USB 3.0 SuperSpeed
[    5.429744] usb usb2: We don't know the algorithms for LPM for this host, disabling LPM.
[    5.436566] hub 2-0:1.0: USB hub found
[    5.444521] hub 2-0:1.0: 1 port detected
[    5.448229] xhci-hcd xhci-hcd.1.auto: xHCI Host Controller
[    5.452060] xhci-hcd xhci-hcd.1.auto: new USB bus registered, assigned bus number 3
[    5.457560] xhci-hcd xhci-hcd.1.auto: hcc params 0x0228f065 hci version 0x100 quirks 0x0000000002010010
[    5.464998] xhci-hcd xhci-hcd.1.auto: irq 44, io mem 0x10000000
[    5.474876] hub 3-0:1.0: USB hub found
[    5.480195] hub 3-0:1.0: 1 port detected
[    5.484247] xhci-hcd xhci-hcd.1.auto: xHCI Host Controller
[    5.488077] xhci-hcd xhci-hcd.1.auto: new USB bus registered, assigned bus number 4
[    5.493375] xhci-hcd xhci-hcd.1.auto: Host supports USB 3.0 SuperSpeed
[    5.501083] usb usb4: We don't know the algorithms for LPM for this host, disabling LPM.
[    5.507889] hub 4-0:1.0: USB hub found
[    5.515889] hub 4-0:1.0: 1 port detected
[    5.520871] kmodloader: done loading kernel modules from /etc/modules-boot.d/*
[    5.524225] init: - preinit -
[    5.854386] s1a: Restricting voltage, 1150000-1102500uV
[    6.034351] s1a: Restricting voltage, 1150000-1102500uV
[    6.054355] s1a: Restricting voltage, 1150000-1102500uV
[    6.104274] s1a: Restricting voltage, 1150000-1102500uV
[    6.175825] s1a: Restricting voltage, 1150000-1102500uV
[    6.364742] s1a: Restricting voltage, 1150000-1102500uV
[    6.467184] s1a: Restricting voltage, 1150000-1102500uV
[    6.549464] s1a: Restricting voltage, 1150000-1102500uV
[    7.305766] s1a: Restricting voltage, 1150000-1102500uV
[    7.423752] s1a: Restricting voltage, 1150000-1102500uV
[    7.446384] Generic PHY fixed-0:01: attached PHY driver [Generic PHY] (mii_bus:phy_addr=fixed-0:01, irq=POLL)
[    7.447540] dwmac1000: Master AXI performs any burst length
[    7.455388] ipq806x-gmac-dwmac 37400000.ethernet eth1: No Safety Features support found
[    7.460668] ipq806x-gmac-dwmac 37400000.ethernet eth1: IEEE 1588-2008 Advanced Timestamp supported
[    7.468867] ipq806x-gmac-dwmac 37400000.ethernet eth1: registered PTP clock
[    7.478202] IPv6: ADDRCONF(NETDEV_UP): eth1: link is not ready
[    8.554278] ipq806x-gmac-dwmac 37400000.ethernet eth1: Link is Up - 1Gbps/Full - flow control off
[    8.554457] IPv6: ADDRCONF(NETDEV_CHANGE): eth1: link becomes ready
[   11.594356] s1a: Restricting voltage, 1150000-1102500uV
[   11.644421] UBIFS (ubi0:1): background thread "ubifs_bgt0_1" started, PID 147
[   11.795960] UBIFS (ubi0:1): UBIFS: mounted UBI device 0, volume 1, name "rootfs_data"
[   11.795983] UBIFS (ubi0:1): LEB size: 126976 bytes (124 KiB), min./max. I/O unit sizes: 2048 bytes/2048 bytes
[   11.802765] UBIFS (ubi0:1): FS size: 89645056 bytes (85 MiB, 706 LEBs), journal size 4444160 bytes (4 MiB, 35 LEBs)
[   11.812721] UBIFS (ubi0:1): reserved for root: 4234156 bytes (4134 KiB)
[   11.822978] UBIFS (ubi0:1): media format: w4/r0 (latest is w5/r0), UUID 32BE8A16-E797-470F-BC0B-F11850EA3266, small LPT model
[   11.844506] mount_root: overlay filesystem has not been fully initialized yet
[   11.844794] mount_root: switching to ubifs overlay
[   11.861242] overlayfs: upper fs does not support xattr, falling back to index=off and metacopy=off.
[   11.925954] urandom-seed: Seed file not found (/etc/urandom.seed)
[   12.003029] s1a: Restricting voltage, 1150000-1102500uV
[   12.074537] procd: - early -
[   12.074695] procd: - watchdog -
[   12.718844] procd: - watchdog -
[   12.719473] procd: - ubus -
[   12.800302] s1a: Restricting voltage, 1150000-1102500uV
[   12.801920] procd: - init -
[   12.834339] s1a: Restricting voltage, 1150000-1102500uV
[   13.136889] kmodloader: loading kernel modules from /etc/modules.d/*
[   13.150949] Loading modules backported from Linux version v5.4-rc8-0-gaf42d3466bdc
[   13.150974] Backport generated by backports.git v5.4-rc8-1-0-g368e8c51
[   13.154487] urngd: v1.0.1 started.
[   13.165559] xt_time: kernel timezone is -0000
[   13.191973] PPP generic driver version 2.4.2
[   13.192791] NET: Registered protocol family 24
[   13.206638] ath10k_pci 0000:01:00.0: assign IRQ: got 35
[   13.206657] ath10k 4.19 driver, optimized for CT firmware, probing pci device: 0x46.
[   13.207034] ath10k_pci 0000:01:00.0: enabling device (0140 -> 0142)
[   13.213515] ath10k_pci 0000:01:00.0: enabling bus mastering
[   13.214002] ath10k_pci 0000:01:00.0: pci irq msi oper_irq_mode 2 irq_mode 0 reset_mode 0
[   13.442589] s1a: Restricting voltage, 1150000-1102500uV
[   13.530859] s1a: Restricting voltage, 1150000-1102500uV
[   13.543460] firmware ath10k!fwcfg-pci-0000:01:00.0.txt: firmware_loading_store: map pages failed
[   13.982271] s1a: Restricting voltage, 1150000-1102500uV
[   14.018880] ath10k_pci 0000:01:00.0: qca9984/qca9994 hw1.0 target 0x01000000 chip_id 0x00000000 sub 168c:cafe
[   14.018913] ath10k_pci 0000:01:00.0: kconfig debug 0 debugfs 1 tracing 0 dfs 1 testmode 0
[   14.029693] ath10k_pci 0000:01:00.0: firmware ver 10.4b-ct-9984-fH-012-6acc9b999 api 5 features mfp,peer-flow-ctrl,txstatus-noack,wmi-10.x-CT,ratemask-CT,regdump-CT,txrate-CT,flush-all-CT,pingpong-CT,ch-regs-CT,nop-CT,htt-mgt-CT,set-special-CT,tx-rc-CT,cust-stats-CT,txrate2-CT,beacon-cb-CT,wmi-block-ack-CT,wmi-bcn-rc-CT crc32 e1a5ec81
[   14.084335] s1a: Restricting voltage, 1150000-1102500uV
[   16.348708] ath10k_pci 0000:01:00.0: board_file api 2 bmi_id 0:1 crc32 85498734
[   22.173195] ath10k_pci 0000:01:00.0: 10.4 wmi init: vdevs: 16  peers: 48  tid: 96
[   22.173222] ath10k_pci 0000:01:00.0: msdu-desc: 2500  skid: 32
[   22.255522] ath10k_pci 0000:01:00.0: wmi print 'P 48/48 V 16 K 144 PH 176 T 186  msdu-desc: 2500  sw-crypt: 0 ct-sta: 0'
[   22.256364] ath10k_pci 0000:01:00.0: wmi print 'free: 81800 iram: 23476 sram: 14440'
[   22.483479] ath10k_pci 0000:01:00.0: htt-ver 2.2 wmi-op 6 htt-op 4 cal pre-cal-file max-sta 32 raw 0 hwcrypto 1
[   22.605615] ath: EEPROM regdomain: 0x0
[   22.605628] ath: EEPROM indicates default country code should be used
[   22.605637] ath: doing EEPROM country->regdmn map search
[   22.605657] ath: country maps to regdmn code: 0x3a
[   22.605670] ath: Country alpha2 being used: US
[   22.605680] ath: Regpair used: 0x3a
[   22.610978] ath10k_pci 0001:01:00.0: assign IRQ: got 37
[   22.611016] ath10k 4.19 driver, optimized for CT firmware, probing pci device: 0x46.
[   22.612188] ath10k_pci 0001:01:00.0: enabling device (0140 -> 0142)
[   22.618118] ath10k_pci 0001:01:00.0: enabling bus mastering
[   22.618909] ath10k_pci 0001:01:00.0: pci irq msi oper_irq_mode 2 irq_mode 0 reset_mode 0
[   22.624266] s1a: Restricting voltage, 1150000-1102500uV
[   22.694424] s1a: Restricting voltage, 1150000-1102500uV
[   23.112971] firmware ath10k!fwcfg-pci-0001:01:00.0.txt: firmware_loading_store: map pages failed
[   23.190931] ath10k_pci 0001:01:00.0: qca9984/qca9994 hw1.0 target 0x01000000 chip_id 0x00000000 sub 168c:cafe
[   23.190964] ath10k_pci 0001:01:00.0: kconfig debug 0 debugfs 1 tracing 0 dfs 1 testmode 0
[   23.201910] ath10k_pci 0001:01:00.0: firmware ver 10.4b-ct-9984-fH-012-6acc9b999 api 5 features mfp,peer-flow-ctrl,txstatus-noack,wmi-10.x-CT,ratemask-CT,regdump-CT,txrate-CT,flush-all-CT,pingpong-CT,ch-regs-CT,nop-CT,htt-mgt-CT,set-special-CT,tx-rc-CT,cust-stats-CT,txrate2-CT,beacon-cb-CT,wmi-block-ack-CT,wmi-bcn-rc-CT crc32 e1a5ec81
[   25.558783] ath10k_pci 0001:01:00.0: board_file api 2 bmi_id 0:2 crc32 85498734
[   31.467503] ath10k_pci 0001:01:00.0: 10.4 wmi init: vdevs: 16  peers: 48  tid: 96
[   31.467532] ath10k_pci 0001:01:00.0: msdu-desc: 2500  skid: 32
[   31.551871] ath10k_pci 0001:01:00.0: wmi print 'P 48/48 V 16 K 144 PH 176 T 186  msdu-desc: 2500  sw-crypt: 0 ct-sta: 0'
[   31.552747] ath10k_pci 0001:01:00.0: wmi print 'free: 81800 iram: 23476 sram: 14440'
[   31.964727] ath10k_pci 0001:01:00.0: htt-ver 2.2 wmi-op 6 htt-op 4 cal pre-cal-file max-sta 32 raw 0 hwcrypto 1
[   32.065811] ath: EEPROM regdomain: 0x0
[   32.065824] ath: EEPROM indicates default country code should be used
[   32.065834] ath: doing EEPROM country->regdmn map search
[   32.065852] ath: country maps to regdmn code: 0x3a
[   32.065866] ath: Country alpha2 being used: US
[   32.065876] ath: Regpair used: 0x3a
[   32.081062] kmodloader: done loading kernel modules from /etc/modules.d/*
[   33.904221] s1a: Restricting voltage, 1150000-1102500uV
[   33.904273] s1a: Restricting voltage, 1150000-1102500uV
[   33.908272] cpufreq: __target_index: Failed to change cpu frequency: -22
[   34.875773] s1a: Restricting voltage, 1150000-1102500uV
[   34.980691] s1a: Restricting voltage, 1150000-1102500uV
[   35.097577] s1a: Restricting voltage, 1150000-1102500uV
[   35.824994] Generic PHY fixed-0:01: attached PHY driver [Generic PHY] (mii_bus:phy_addr=fixed-0:01, irq=POLL)
[   35.825649] dwmac1000: Master AXI performs any burst length
[   35.833890] ipq806x-gmac-dwmac 37400000.ethernet eth1: No Safety Features support found
[   35.839348] ipq806x-gmac-dwmac 37400000.ethernet eth1: IEEE 1588-2008 Advanced Timestamp supported
[   35.847432] ipq806x-gmac-dwmac 37400000.ethernet eth1: registered PTP clock
[   35.856436] IPv6: ADDRCONF(NETDEV_UP): eth1: link is not ready
[   35.866425] br-lan: port 1(eth1.1) entered blocking state
[   35.869038] br-lan: port 1(eth1.1) entered disabled state
[   35.874827] device eth1.1 entered promiscuous mode
[   35.879886] device eth1 entered promiscuous mode
[   35.885786] IPv6: ADDRCONF(NETDEV_UP): br-lan: link is not ready
[   35.897882] Generic PHY fixed-0:00: attached PHY driver [Generic PHY] (mii_bus:phy_addr=fixed-0:00, irq=POLL)
[   35.898597] dwmac1000: Master AXI performs any burst length
[   35.906879] ipq806x-gmac-dwmac 37200000.ethernet eth0: No Safety Features support found
[   35.912158] ipq806x-gmac-dwmac 37200000.ethernet eth0: IEEE 1588-2008 Advanced Timestamp supported
[   35.920316] ipq806x-gmac-dwmac 37200000.ethernet eth0: registered PTP clock
[   35.929535] IPv6: ADDRCONF(NETDEV_UP): eth0: link is not ready
[   35.940128] IPv6: ADDRCONF(NETDEV_UP): eth0.3: link is not ready
[   35.943172] IPv6: ADDRCONF(NETDEV_UP): eth0.6: link is not ready
[   36.525031] ath: EEPROM regdomain: 0x82d4
[   36.525050] ath: EEPROM indicates we should expect a country code
[   36.528013] ath: doing EEPROM country->regdmn map search
[   36.534169] ath: country maps to regdmn code: 0x37
[   36.539470] ath: Country alpha2 being used: ES
[   36.544148] ath: Regpair used: 0x37
[   36.548498] ath: regdomain 0x82d4 dynamically updated by user
[   36.551905] ath: EEPROM regdomain: 0x82d4
[   36.557863] ath: EEPROM indicates we should expect a country code
[   36.561780] ath: doing EEPROM country->regdmn map search
[   36.567922] ath: country maps to regdmn code: 0x37
[   36.573236] ath: Country alpha2 being used: ES
[   36.577886] ath: Regpair used: 0x37
[   36.582263] ath: regdomain 0x82d4 dynamically updated by user
[   36.874236] ipq806x-gmac-dwmac 37400000.ethernet eth1: Link is Up - 1Gbps/Full - flow control off
[   36.954186] ipq806x-gmac-dwmac 37200000.ethernet eth0: Link is Up - 1Gbps/Full - flow control off
[   42.780961] ath10k_pci 0001:01:00.0: 10.4 wmi init: vdevs: 16  peers: 48  tid: 96
[   42.780990] ath10k_pci 0001:01:00.0: msdu-desc: 2500  skid: 32
[   42.865016] ath10k_pci 0001:01:00.0: wmi print 'P 48/48 V 16 K 144 PH 176 T 186  msdu-desc: 2500  sw-crypt: 0 ct-sta: 0'
[   42.865877] ath10k_pci 0001:01:00.0: wmi print 'free: 81800 iram: 23476 sram: 14440'
[   43.335929] ath10k_pci 0001:01:00.0: Firmware lacks feature flag indicating a retry limit of > 2 is OK, requested limit: 4
[   43.336206] IPv6: ADDRCONF(NETDEV_UP): wlan1: link is not ready
[   43.346593] IPv6: ADDRCONF(NETDEV_CHANGE): eth0: link becomes ready
[   43.352305] IPv6: ADDRCONF(NETDEV_CHANGE): eth0.3: link becomes ready
[   43.358352] IPv6: ADDRCONF(NETDEV_CHANGE): eth0.6: link becomes ready
[   43.364926] IPv6: ADDRCONF(NETDEV_CHANGE): eth1: link becomes ready
[   43.371810] br-lan: port 1(eth1.1) entered blocking state
[   43.377098] br-lan: port 1(eth1.1) entered forwarding state
[   43.383069] br-lan: port 2(wlan1) entered blocking state
[   43.388142] br-lan: port 2(wlan1) entered disabled state
[   43.393736] device wlan1 entered promiscuous mode
[   43.400636] IPv6: ADDRCONF(NETDEV_CHANGE): br-lan: link becomes ready
[   43.420677] ath: EEPROM regdomain: 0x8348
[   43.420699] ath: EEPROM indicates we should expect a country code
[   43.423661] ath: doing EEPROM country->regdmn map search
[   43.429908] ath: country maps to regdmn code: 0x3a
[   43.435191] ath: Country alpha2 being used: US
[   43.439718] ath: Regpair used: 0x3a
[   43.444220] ath: regdomain 0x8348 dynamically updated by user
[   43.447587] ath: EEPROM regdomain: 0x8348
[   43.453440] ath: EEPROM indicates we should expect a country code
[   43.457497] ath: doing EEPROM country->regdmn map search
[   43.463505] ath: country maps to regdmn code: 0x3a
[   43.468962] ath: Country alpha2 being used: US
[   43.473486] ath: Regpair used: 0x3a
[   43.477992] ath: regdomain 0x8348 dynamically updated by user
[   43.779733] ath10k_pci 0001:01:00.0: NOTE:  Firmware DBGLOG output disabled in debug_mask: 0x10000000
[   46.830367] ath10k_pci 0001:01:00.0: mac-vif-chan had error in htt_rx_h_vdev_channel, peer-id: 0  vdev-id: 0 peer-addr: b0:7f:b9:3e:2e:bc.
[   49.706820] ath10k_pci 0000:01:00.0: 10.4 wmi init: vdevs: 16  peers: 48  tid: 96
[   49.706846] ath10k_pci 0000:01:00.0: msdu-desc: 2500  skid: 32
[   49.789101] ath10k_pci 0000:01:00.0: wmi print 'P 48/48 V 16 K 144 PH 176 T 186  msdu-desc: 2500  sw-crypt: 0 ct-sta: 0'
[   49.789953] ath10k_pci 0000:01:00.0: wmi print 'free: 81800 iram: 23476 sram: 14440'
[   49.924320] s1a: Restricting voltage, 1150000-1102500uV
[   49.924365] s1a: Restricting voltage, 1150000-1102500uV
[   49.928362] cpufreq: __target_index: Failed to change cpu frequency: -22
[   50.064238] s1a: Restricting voltage, 1150000-1102500uV
[   50.064284] s1a: Restricting voltage, 1150000-1102500uV
[   50.068283] cpufreq: __target_index: Failed to change cpu frequency: -22
[   50.114277] s1a: Restricting voltage, 1150000-1102500uV
[   50.114323] s1a: Restricting voltage, 1150000-1102500uV
[   50.118323] cpufreq: __target_index: Failed to change cpu frequency: -22
[   50.152598] ath10k_pci 0000:01:00.0: Firmware lacks feature flag indicating a retry limit of > 2 is OK, requested limit: 4
[   50.152769] IPv6: ADDRCONF(NETDEV_UP): wlan0: link is not ready
[   50.164160] pppoe-wan: renamed from ppp0
[   50.217568] br-lan: port 3(wlan0) entered blocking state
[   50.217599] br-lan: port 3(wlan0) entered disabled state
[   50.222153] device wlan0 entered promiscuous mode
[   50.245924] ath: EEPROM regdomain: 0x82d4
[   50.245945] ath: EEPROM indicates we should expect a country code
[   50.248908] ath: doing EEPROM country->regdmn map search
[   50.255085] ath: country maps to regdmn code: 0x37
[   50.260367] ath: Country alpha2 being used: ES
[   50.265069] ath: Regpair used: 0x37
[   50.269394] ath: regdomain 0x82d4 dynamically updated by user
[   50.272827] ath: EEPROM regdomain: 0x82d4
[   50.278752] ath: EEPROM indicates we should expect a country code
[   50.282677] ath: doing EEPROM country->regdmn map search
[   50.288843] ath: country maps to regdmn code: 0x37
[   50.294185] ath: Country alpha2 being used: ES
[   50.298736] ath: Regpair used: 0x37
[   50.303160] ath: regdomain 0x82d4 dynamically updated by user
[   50.744252] s1a: Restricting voltage, 1150000-1102500uV
[   50.744301] s1a: Restricting voltage, 1150000-1102500uV
[   50.748300] cpufreq: __target_index: Failed to change cpu frequency: -22
[   50.794243] s1a: Restricting voltage, 1150000-1102500uV
[   50.794289] s1a: Restricting voltage, 1150000-1102500uV
[   50.798284] cpufreq: __target_index: Failed to change cpu frequency: -22
[   50.971030] IPv6: ADDRCONF(NETDEV_CHANGE): wlan0: link becomes ready
[   50.971240] br-lan: port 3(wlan0) entered blocking state
[   50.971771] s1a: Restricting voltage, 1150000-1102500uV
[   50.976533] br-lan: port 3(wlan0) entered forwarding state
[   51.906129] ath10k_pci 0000:01:00.0: Invalid peer id 0 or peer stats buffer, peer: e3a3a1a9  sta:   (null)
[   52.054229] s1a: Restricting voltage, 1150000-1102500uV
[   52.054282] s1a: Restricting voltage, 1150000-1102500uV
[   52.058441] cpufreq: __target_index: Failed to change cpu frequency: -22
[   53.914352] s1a: Restricting voltage, 1150000-1102500uV
[   53.914408] s1a: Restricting voltage, 1150000-1102500uV
[   53.918406] cpufreq: __target_index: Failed to change cpu frequency: -22
[   53.964238] s1a: Restricting voltage, 1150000-1102500uV
[   53.964285] s1a: Restricting voltage, 1150000-1102500uV
[   53.968282] cpufreq: __target_index: Failed to change cpu frequency: -22
[   55.219062] s1a: Restricting voltage, 1150000-1102500uV
[   55.266135] s1a: Restricting voltage, 1150000-1102500uV
[   57.500153] IPv6: ADDRCONF(NETDEV_CHANGE): wlan1: link becomes ready
[   57.500556] br-lan: port 2(wlan1) entered blocking state
[   57.505746] br-lan: port 2(wlan1) entered forwarding state
[   63.525703] s1a: Restricting voltage, 1150000-1102500uV
[  120.924883] s1a: Restricting voltage, 1150000-1102500uV
[  121.010809] s1a: Restricting voltage, 1150000-1102500uV
[  121.365736] s1a: Restricting voltage, 1150000-1102500uV
[  121.461169] s1a: Restricting voltage, 1150000-1102500uV
[  121.461212] s1a: Restricting voltage, 1150000-1102500uV
[  121.465447] cpufreq: __target_index: Failed to change cpu frequency: -22
[  128.825580] s1a: Restricting voltage, 1150000-1102500uV
[  128.825632] s1a: Restricting voltage, 1150000-1102500uV
[  128.829813] cpufreq: __target_index: Failed to change cpu frequency: -22
[  134.396063] s1a: Restricting voltage, 1150000-1102500uV
[  134.845783] s1a: Restricting voltage, 1150000-1102500uV
[  134.845838] s1a: Restricting voltage, 1150000-1102500uV
[  134.849963] cpufreq: __target_index: Failed to change cpu frequency: -22
[  134.895801] s1a: Restricting voltage, 1150000-1102500uV
[  134.895846] s1a: Restricting voltage, 1150000-1102500uV
[  134.900017] cpufreq: __target_index: Failed to change cpu frequency: -22
[  137.646081] s1a: Restricting voltage, 1150000-1102500uV
[  178.659676] s1a: Restricting voltage, 1150000-1102500uV
[  294.144436] s1a: Restricting voltage, 1150000-1102500uV
[  329.845269] s1a: Restricting voltage, 1150000-1102500uV
[  459.787416] s1a: Restricting voltage, 1150000-1102500uV
[  460.277411] s1a: Restricting voltage, 1150000-1102500uV
[  461.947494] s1a: Restricting voltage, 1150000-1102500uV
[  462.047400] s1a: Restricting voltage, 1150000-1102500uV
[  462.137467] s1a: Restricting voltage, 1150000-1102500uV
[  462.317373] s1a: Restricting voltage, 1150000-1102500uV
[  462.937429] s1a: Restricting voltage, 1150000-1102500uV
[  463.087289] s1a: Restricting voltage, 1150000-1102500uV
[  463.087333] s1a: Restricting voltage, 1150000-1102500uV
[  463.091475] cpufreq: __target_index: Failed to change cpu frequency: -22
[  463.597431] s1a: Restricting voltage, 1150000-1102500uV
[  549.657668] s1a: Restricting voltage, 1150000-1102500uV
[  555.097621] s1a: Restricting voltage, 1150000-1102500uV
[  560.087722] s1a: Restricting voltage, 1150000-1102500uV
[  561.727596] s1a: Restricting voltage, 1150000-1102500uV
[  561.967480] s1a: Restricting voltage, 1150000-1102500uV
[  562.977602] s1a: Restricting voltage, 1150000-1102500uV`

Ansuel committed:
ipq806x: refactor l2 freq scaling patch

Refactor l2 freq scaling patch to support voltage
scaling and add support to base cache scaling on
cpu freq scaling. Update the dtsi files with the new
definition used in the new code.

Signed-off-by: Ansuel Smith <ansuelsmth@gmail.com>

Ansuel commented:
@Noltari try with this version ? Think the problem is with voltage tollerance and the regulator is set below the min microvolt...

shelterx commented:
Just updated, works for me anyway. Nothing odd in the logs.

Noltari commented:
> @Noltari try with this version ? Think the problem is with voltage tollerance and the regulator is set below the min microvolt...

@Ansuel no errors on that version (it's been running for 10h now).

ynezz commented:
@Noltari Can you handle this? Thanks.

Noltari commented:
@ynezz sure.
@Ansuel merged, thanks!
